lz4-compress/decompress.rs

```diff
@@ -0,0 +1,8 @@
+#![no_main]
+
+#[macro_use] extern crate libfuzzer_sys;
+extern crate lz4_compress;
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(_) = lz4_compress::decompress(data) {}
```

Comment:
Is there something wrong with doing:

```rust
let _ = lz4_compress::decompress(data);
```

Is there a concern it'd be optimized away?